workers/unity/Assets/Tests/PlaymodeTests/Performance/ComponentUpdateTests.cs

```diff
@@ -0,0 +1,51 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.TestUtils;
+using Improbable.Worker.CInterop;
+using NUnit.Framework;
+using Unity.PerformanceTesting;
+
+namespace Improbable.Gdk.PlaymodeTests
+{
+    [TestFixture]
+    public class ComponentUpdateTests : MockBase
+    {
+        private const string WorkerType = "TestWorkerType";
+        private const long EntityId = 100;
+
+        [Performance, Test]
+        public void SinglePositionUpdate()
+        {
+            var markers = new[] { "GatherAllChunks", "ExecuteReplication", "Position.SendUpdates" };
+
+            var currentState = World.Step(world =>
+            {
+                world.Connection.CreateEntity(EntityId, GetEntityTemplate());
+                world.Connection.ChangeAuthority(EntityId, Position.ComponentId, Authority.Authoritative);
+            });
+
+            ActionMeasurement.Measure(() =>
+                {
+                    currentState.Step(world =>
+                    {
+                        var workerSystem = world.Worker.World.GetExistingSystem<WorkerSystem>();
```

Comment:
We can probably add some helpers for this: 

```
var workerSystem = world.System<WorkerSystem>();
```

to make it more brief

Reply:
we already had a `GetSystem` helper, I just didn't use it 😬